GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -125,7 +125,7 @@ function CreateMetaInstaller()
     echo "$mountCmd"
     eval $mountCmd || exit 1
     
-    MOUNTEDVOLUME=`/usr/bin/find /Volumes -type d -name "Git $GITVERSIONSTRING*"`
+    MOUNTEDVOLUME=`/usr/bin/find /Volumes -maxdepth 1 -type d -name "Git $GITVERSIONSTRING*"`
```

Comment:
Do you know why we need to run find at all?  Does the script not know where to expect the installer? 

cc: @alameenshah

Reply:
The naming convention of the downloaded .nupkg, the .dmg inside of it, the name of the volume of the .dmg will mount and the .pkg that we'll actually install/bundle is not consistent. 

We have a bunch of code like this where we have to use find to figure out what the exact name/path of the next phase of our pipeline is going to be.

From the build script:
```
echo 'Downloading a VFS-enabled version of Git...'
$VFS_SCRIPTDIR/DownloadGVFSGit.sh || exit 1
GITVERSION="$($VFS_SCRIPTDIR/GetGitVersionNumber.sh)"
GITPATH="$(find $VFS_PACKAGESDIR/gitformac.gvfs.installer/$GITVERSION -type f -name *.dmg)" || exit 1
echo "Downloaded Git $GITVERSION"
# Now that we have a path containing the version number, generate GVFSConstants.GitVersion.cs
$VFS_SCRIPTDIR/GenerateGitVersionConstants.sh "$GITPATH" $BUILDDIR || exit 1
```

In the future, I'd love to rework how the Git installer works to reduce some of the complexity here.